### PR TITLE
Inflight Fuel switcher

### PR DESCRIPTION
**Pull request by viseriod**
Ironed out the bugs for inflight. Should work for all engines just need to update the configs and use EngineECU2 as a base.

### CODE REVIEW

File: FNPlugin/Propulsion/EngineECU2.cs [hunk 1]
```diff
@@ -95,8 +91,8 @@ public IList<FuelConfiguration> FuelConfigurations
             {
                 if (fuelConfigurations == null)
                 {
-                    fuelConfigurations = part.FindModulesImplementing<FuelConfiguration>().OrderByDescending(f => f.maxIsp).ToList();
-                        
+                    fuelConfigurations = part.FindModulesImplementing<FuelConfiguration>().OrderByDescending(f => f.maxThrust).ToList();
```
Comment (viseriod) [inline FNPlugin/Propulsion/EngineECU2.cs]: figured that most people want highest thrust listed first

File: GameData/WarpPlugin/Parts/Engines/vista/Vista.cfg [hunk 2]
```diff
@@ -128,10 +128,10 @@ emissiveConstant = 0.95
     	fuelConfigurationName = D-T
     	fuels = LqdHydrogen, LqdDeuterium, LqdTritium
     	ratios = 1.38337941, 1.38337941e-3, 1.38337941e-3
-    	amount = 20000, 40, 40
-    	maxAmount = 20000, 40, 40
+    	amount = 20000, 1000, 1000
+    	maxAmount = 20000, 1000, 1000
     	maxIsp = 15500
-    	thrustMult = 1
```
Comment (viseriod) [inline GameData/WarpPlugin/Parts/Engines/vista/Vista.cfg]: default is 1 so no need for this line

**Comment by sswelm** [comment]
Alright, it works but you still need to take care of some edge cases like
where you provide it with insufficient power, it should not send you to the
void

2017-02-24 7:37 GMT+01:00 viseriod <notifications@github.com>:

**Comment by sswelm** [comment]
sorry, I wasn't clear enough, provide it with insufficient power and switch
mode will send you to the void

2017-02-24 17:24 GMT+01:00 Sebastiaan Willem van Swelm <sswelm@gmail.com>:

**Comment by sswelm** [comment]
Alright, it works but you still need to take care of some edge cases like
where you provide it with insufficient power and switch fuel mode, it
should not send you to the void

2017-02-24 17:26 GMT+01:00 Sebastiaan Willem van Swelm <sswelm@gmail.com>:

**Pull request merged**